public/index.html

```
@@ -5,202 +5,791 @@
-    <select id="filter-format"
-      class="px-4 py-2.5 rounded-xl border border-slate-200 bg-white shadow-sm text-sm focus:outline-none focus:ring-2 focus:ring-brand/40">
-      <option value="">All Formats</option>
-      <option value="live">🎥 Live</option>
-      <option value="self_paced">⏱ Self-paced</option>
-      <option value="hybrid">🔀 Hybrid</option>
-    </select>
-  </div>
+<!-- ════════════════════════════════════ ACTIVITY BROWSER ══ -->
+<section style="max-width:1200px;margin:5rem auto;padding:0 1.5rem;" id="activities">
+
+  <div style="display:flex;align-items:flex-end;justify-content:space-between;margin-bottom:2rem;flex-wrap:wrap;gap:1rem;">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </div> ], start tag match failed [ <div style="display:flex;align-items:flex-end;justify-content:space-between;margin-bottom:2rem;flex-wrap:wrap;gap:1rem;"> ] on line 464.

(tag-pair)